src/Form.ts

```diff
@@ -59,16 +59,22 @@ export default class Form {
       handleErrors: action,
       isComplete: computed,
       values: computed,
-      isDirty: computed
+      isDirty: computed,
+      serialized: computed
     })
   }
 
-  data(): Values {
+  get serialized(): Values {
     return flat.unflatten(
-      mapValues(this.fields, (field: Field) => field._mapOut())
+      mapValues(this.fields, (field: field) => field._mapOut())
     )
   }
 
+  // TODO: Deprecate
+  data(): Values {
+    return this.serialized
```

Comment:
Maybe a console log here with a deprecation warning

Reply:
loool andreees

Reply:
Ey! Es Open Source xD

Reply:
@andresgutgon wtf

Reply:
welcome back @andresgutgon

Reply:
Just to complement what @andresgutgon said, you can also add a JSDoc kind of comment to mark it as deprecated to code editors will highlight it in a special way (i.e. vscode do a strikethrough)

```ts
/**
  @deprecated
*/
data(): Values {
```